ios/RCTPSPDFKit/RCTPSPDFKitView.m

```diff
@@ -234,8 +240,14 @@ - (BOOL)removeAnnotationWithUUID:(NSString *)annotationUUID {
 }
 
 - (NSDictionary<NSString *, NSArray<NSDictionary *> *> *)getAllUnsavedAnnotations {
-  PSPDFDocumentProvider *documentProvider = self.pdfController.document.documentProviders.firstObject;
-  NSData *data = [self.pdfController.document generateInstantJSONFromDocumentProvider:documentProvider error:NULL];
+  PSPDFDocument *document = self.pdfController.document;
+  if (!document.isValid) {
+    NSLog(@"Document is invalid.");
```

Comment:
can we macro that?

Reply:
Great idea! Done in https://github.com/PSPDFKit/react-native/pull/210/commits/24c34b202ecba86f47af0e0b0dde85f07c3c0401